### PR TITLE
MEN-4557: Documentation for Update Control configuration options

### PR DESCRIPTION
**Pull request by alfrunes**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Thinking out loud: should we put this into a feature branch for now? These days we expect small docs fixes here and there that we want to release to `hosted` branch, but documenting a not yet released feature like this PR will prevent us to do simple merges.

I know is a headache to maintain a feature branch in this repo, but we could at least keep it on a feature branch for one week or something.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 1]
```diff
@@ -237,4 +237,16 @@ The location where to store the device_type. This must be on a persistent
...
 
+#### UpdateControlMapExipreTimeSeconds
 
+The expire time in seconds for an update control map. The value provides the
+time window for an application using the update control _dbus API_ to provide
+input before the update control map to enter the expired state. _Defaults to
+2*UpdatePollIntervalSeconds_
+<!-- TODO: Insert links to dbus api specs for update control -->
+
+#### UpdateControlMapBootExpireTimeSeconds
```
Comment (kacf) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:248]: Naming, see above.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 2]
```diff
@@ -237,4 +237,16 @@ The location where to store the device_type. This must be on a persistent
...
+
+The maximum expire time in seconds after startup for an update control map. The
+value provides an upper bound on the update control map [expire time](#UpdateControlMapExipreTimeSeconds) after
```
Comment (kacf) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:251]: Same typo.

**Review by kacf — CHANGES_REQUESTED** [review]
> Thinking out loud: should we put this into a feature branch for now?

Yes, please do. Most tickets have this in their criteria, but looks like I forgot it in this ticket. I have added it now.

File: 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md [hunk 3]
```diff
@@ -237,4 +237,17 @@ The location where to store the device_type. This must be on a persistent
 partition to avoid it accidentally changing due to a root filesystem update. The
 default location is `/var/lib/mender/device_type`
 
+#### UpdateControlMapExpirationTimeSeconds
 
+The expire time in seconds for an update control map. The value provides the
+time window for an application using the update control _dbus API_ to refresh
+the update control map to enter the expired state. _Defaults to
```
Comment (kacf) [inline 03.Client-installation/06.Configuration-file/50.Configuration-options/docs.md:244]: Isn't this sentence a bit weird? Maybe "...to refresh the update control map **before it enters** the expired state."

**Review by kacf — APPROVED** [review]
Is the sentence below a bit weird? Fix if you agree, if not then go ahead!

**Pull request merged**